### DISCUSSION TITLE
Evernote Importer: More clear surfacing of logs, errors

### DISCUSSION THREAD

**Discussion by pbryant-phew** [standardnotes/forum#2942] [Suggestions]
**Have you read the [Longevity Statement](https://standardnotes.org/longevity)?**
Yes

**Is your feature request related to a problem? Please describe.**
The Evernote import tool does not, as far as I can see or find in any documentation, produce useful logs of its activities.  I'm particularly interested in error logs, but full logs including success cases would be even more useful.  When importing dozens/hundreds/thousands of Evernote notes, with the goal of permanently deleting those notes from Evernote, it is quite fundamental to know that 100% of those notes have not suffered any processing error in the SN .enex file importer and hence been lost or significantly mangled.  

Notably, the Evernote user often can't know how many notes are in their .enex file (at least not without opening it in a text parser and doing some maths, but your average user isn't capable of doing that and shouldn't be expected to).  This is particularly true when exporting a large Evernote notebook that is split into smaller .enex files, as the SN Evernote import tool guidances suggests you do. So the use has no way of knowing how many notes are _supposed_ to arrive in their SN Notes view.  Furthermore, since the SN Notes view and #evernote tag do not distinguish one Evernote import from another, the SN user has no real opportunity to do maths on the item counts in those SN locations/lists to work out whether some notes perhaps failed to import.  This process is all quite opaque currently, and I would have thought that users wanting to migrate from Evernote was a key source of new users (and hence longevity).

**Describe the solution you'd like**
The importer could offer display/download of a log file providing clear status for the success or otherwise of the import of each note in the source .enex file.

**Describe alternatives you've considered**
Described above, but they are not reasonable or workable alternatives.

**Additional context**
I have searched for other feature requests relating to this but couldn't find any.  Someone will correct me if I've missed that.

**Comment by ILoveSkilledCoders** [comment]
Cool that you want to leave EVN too. I'm not concerned about your issues per say, but to import as many notebooks & make sure ALL of the notes import under the notebooks nicely is a concern of mine. Have you tried the import feature testing with just one notebook?

I'm here to ask a few questions.

I'm not seeing notebooks/folders in Standard Notes. I only see tags & I don't like tags, plus I find them somewhat confusing.

What are your thoughts on that?

I'm shocked that so many of these note taking programs don't have folders/notebooks. SMH

Thanks
